tests/com.espressif.idf.ui.test/src/com/espressif/idf/ui/test/executable/cases/project/NewEspressifIDFProjectSBOMTest.java

```diff
@@ -0,0 +1,216 @@
+		}
+
+		private static void givenProjectNameIs(String projectName)
+		{
+			Fixture.projectName = projectName;
+		}
+
+		private static void whenNewProjectIsSelected() throws Exception
+		{
+			ProjectTestOperations.setupProject(projectName, category, subCategory, bot);
+		}
+
+		private static void whenProjectIsBuiltUsingContextMenu() throws IOException
+		{
+			ProjectTestOperations.buildProjectUsingContextMenu(projectName, bot);
+			ProjectTestOperations.waitForProjectBuild(bot);
+			TestWidgetWaitUtility.waitForOperationsInProgressToFinishAsync(bot);
+		}
+
+		private static void cleanTestEnv()
+		{
+			TestWidgetWaitUtility.waitForOperationsInProgressToFinishAsync(bot);
+			ProjectTestOperations.closeAllProjects(bot);
+			ProjectTestOperations.deleteAllProjects(bot);
+		}
+
+		private static void whenOpenSBOMtool() throws IOException
+		{
+			ProjectTestOperations.launchCommandUsingContextMenu(projectName, bot, "SBOM Tool");
+		}
+
+		private static void thenRunOkbuttonDisabled(SWTWorkbenchBot bot) throws IOException
+		{
+			SWTBotButton okButton = bot.button("OK");
+			assertTrue(!okButton.isEnabled());
+			bot.checkBox("Redirect output to the file").click();
+			assertTrue(!okButton.isEnabled());
+			bot.checkBox("Redirect output to the file").click();
+			assertTrue(!okButton.isEnabled());
+			bot.button("Cancel").click();
+		}
+
+		private static void whenRunSBOMtool() throws IOException
+		{
+			ProjectTestOperations.launchCommandUsingContextMenu(projectName, bot, "SBOM Tool");
+			bot.button("OK").click();
+			ProjectTestOperations
+					.joinJobByName(com.espressif.idf.ui.update.Messages.SbomCommandDialog_EspIdfSbomJobName);
+		}
+
+		private static void thenCheckResultInConsole() throws IOException
+		{
+			ProjectTestOperations.findInConsole(bot, "Espressif IDF Tools Console", "SPDXVersion");
+		}
+
+		private static void thenVerifyConsole() throws IOException
+		{
+			ProjectTestOperations.findInConsole(bot, "Espressif IDF Tools Console",
+					"The output was redirected to the file:");
+		}
+
+		private static void whenRunSBOMtoolUsingContextMenuRedirectOutputToFile() throws IOException
+		{
+			ProjectTestOperations.launchCommandUsingContextMenu(projectName, bot, "SBOM Tool");
+			bot.checkBox("Redirect output to the file").click();
+			bot.button("OK").click();
+			ProjectTestOperations
+					.joinJobByName(com.espressif.idf.ui.update.Messages.SbomCommandDialog_EspIdfSbomJobName);
+		}
+
+		private static void thenCheckResultInFile() throws IOException
+		{
+			ProjectTestOperations.findInConsole(bot, "Espressif IDF Tools Console",
+					"The output was redirected to the file:");
+			ProjectTestOperations.launchCommandUsingContextMenu(projectName, bot, "Refresh");
+			bot.tree().getTreeItem(projectName).getNode("sbom.txt").select();
+			bot.tree().getTreeItem(projectName).getNode("sbom.txt").doubleClick();
+			assertTrue(ProjectTestOperations.checkTextEditorContentForPhrase("SPDXVersion", bot));
```

Comment:
I am not sure if this is correct way to verify the results please consult with someone from ESP-IDF to see that what result should we expect and what can be consistent to verify this. SPDXVersion may also be present in the file if there were some issues in sbom generation. This needs some verification? I think ideal way would be to have the exact file in our resources that we are expecting to be the output and compare those two files.